### ISSUE TITLE
canPlaceMarginOrder

### ISSUE THREAD

**Issue by YetAnotherDima** [mzheravin/exchange-core#74]
RiskEngine.java line 530, probably bug: freeMargin = position.estimateProfit(spec, lastPriceCache.get(spec.symbolId)); 

freeMargin += position.estimateProfit(spec, lastPriceCache.get(spec.symbolId)); ?

**Comment by mzheravin** [comment]
Yes absolutely, this is a bug affecting margin trading. Thanks for pointing it out.

**Comment by bob-dong-20230331** [comment]
yes, it's a bug, and no one fix it....

**Comment by dpy1123** [comment]
yes, it's a bug. and no one fix it....
